project/build.scala

```diff
@@ -1,147 +1,20 @@
 import sbt._
-import Keys._
-import org.scalatra.sbt._
-import org.scalatra.sbt.PluginKeys._
-import com.socrata.cloudbeessbt.SocrataCloudbeesSbt
-import com.mojolly.scalate.ScalatePlugin._
-import ScalateKeys._
 
-import sbtassembly.Plugin.AssemblyKeys._
-import sbtassembly.Plugin.MergeStrategy
-import sbtassembly.Plugin.PathList
-import sbtassembly.AssemblyUtils._
+object Build extends sbt.Build {
+  lazy val geospace= Project( "geospace-proj", file(".") ).
+    settings(BuildSettings.buildSettings : _*).
+    aggregate(geoLibrary, geospaceHttp)
 
-object BuildParameters {
-  val Organization = "com.socrata"
-  val Name = "geospace"
-  val ScalaVersion = "2.10.3"
-  val ScalatraVersion = "2.2.2"
-  val port = SettingKey[Int]("port")
-  val Conf = config("container")
-}
-
-object Dependencies {
-  import BuildParameters._
-
-  lazy val socrataResolvers = Seq(
-    "Open Source Geospatial Foundation Repository" at "http://download.osgeo.org/webdav/geotools",
-    "spray repo" at "http://repo.spray.io",
-    "velvia maven" at "http://dl.bintray.com/velvia/maven"
-  )
+  // originally had aggregate(allOtherProjects: _*) but gave an error
+  // [error] java.lang.IllegalAccessException: Class Build$$anonfun$allOtherProjects$2 can not access a member of class Build$ with modifiers "private"
+  // adding projects directly helps, but to determine cause.
```

Comment:
If that pattern never existed in the code for you to remove, I don't think you need to include this comment.